lint.sh

```diff
@@ -27,9 +27,9 @@ run_cargo_clippy() {
 run_prettier() {
     MODE=$1
     if [ "$MODE" == "check" ]; then
-        prettier -c .prettierrc --check "**/*.$FILE_TYPES"
+        npx --yes prettier@3.2.5 -c .prettierrc --check "**/*.$FILE_TYPES"
```

Comment:
adding version to scripts will not allow renovate to open update PRs

Reply:
we can add a package.json on the project root to lock the npm dependency, in this way renovate will be able to update it but still users will be able to use the expected version

Reply:
any progress on this?

Reply:
The last update was [this suggestion](https://github.com/tailcallhq/tailcall/issues/1099#issuecomment-1937145466), but @neo773 tried, and I think he had some issues with node dependencies on Nix.